http-handler/src/main/java/com/flipkart/poseidon/handlers/http/impl/HttpConnectionPool.java

```diff
@@ -16,12 +16,33 @@
 
 package com.flipkart.poseidon.handlers.http.impl;
 
-import com.flipkart.poseidon.handlers.http.HttpDelete;
-import org.apache.http.*;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.Header;
+import org.apache.http.HeaderElement;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpException;
+import org.apache.http.HttpHost;
+import org.apache.http.HttpRequest;
+import org.apache.http.HttpRequestInterceptor;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpResponseInterceptor;
+import org.apache.http.NameValuePair;
 import org.apache.http.client.HttpClient;
 import org.apache.http.client.entity.GzipCompressingEntity;
 import org.apache.http.client.entity.GzipDecompressingEntity;
-import org.apache.http.client.methods.*;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
```

Comment:
imports are better understood with '*' that is why I had done that explicitly.
http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

anyhow, there is no problem in changing back, will do that :)